planemo/virtualenv.py

```diff
@@ -3,47 +3,28 @@
 from __future__ import absolute_import
 
 import os
-import sys
 
-import virtualenv
 from galaxy.util.commands import which
 
-
 DEFAULT_PYTHON_VERSION = os.environ.get("PLANEMO_DEFAULT_PYTHON_VERSION", "3")
 
 
-def create_and_exit(virtualenv_path, **kwds):
```

Comment:
This function is still used in `planemo/commands/cmd_virtualenv.py` , did you intend to remove that too?